lib/matplotlib/widgets.py

```diff
@@ -514,53 +516,45 @@ class CheckButtons(AxesWidget):
     """
     A GUI neutral set of check buttons.
 
-    For the check buttons to remain responsive you must keep a
-    reference to this object.
+    For the check buttons to remain responsive you must keep a reference to
+    this object.
 
-    The following attributes are exposed
+    Connect to the CheckButtons with the :meth:`on_clicked` method.
 
-     *ax*
-        The :class:`matplotlib.axes.Axes` instance the buttons are
-        located in
+    Attributes
+    ----------
+     ax
+        The :class:`matplotlib.axes.Axes` instance the buttons are located in
 
-     *labels*
+     labels
         List of :class:`matplotlib.text.Text` instances
 
-     *lines*
+     lines
         List of (line1, line2) tuples for the x's in the check boxes.
         These lines exist for each box, but have ``set_visible(False)``
         when its box is not checked.
 
      *rectangles*
         List of :class:`matplotlib.patches.Rectangle` instances
-
-    Connect to the CheckButtons with the :meth:`on_clicked` method
     """
-    def __init__(self, ax, labels, actives=None):
```

Comment:
Please leave as is. You mustn't remove the default value. Also, renaming a parameter is a backward-incompatible API change. That would need a deprecation period. I think  that's not worth the effort, but if you really want to change it, add a deprecation note to next_api_changes. Since recently, we have a decorator that can handle renaming of parameters with deprecation logic, which would have to be added to the code.